AirCasting/SessionViews/ChartEntriesCreator.swift

```diff
@@ -0,0 +1,47 @@
+// Created by Lunar on 05/05/2021.
+//
+
+import Foundation
+import Charts
+
+final class ChartEntriesCreator {
+    var entries: [ChartDataEntry] = []
+    var stream: MeasurementStreamEntity
+    lazy var timeUnit: Double = stream.session.type == .mobile ? 60 : 60*60
+    lazy var lastMeasurementTime = {
+        self.stream.lastMeasurementTime
+    }
+    
+    init(stream: MeasurementStreamEntity) {
+        self.stream = stream
+    }
+    
+    func generateEntries() -> [ChartDataEntry] {
+        guard let lastMeasurementTime = lastMeasurementTime() else {
+            return []
+        }
+        
+        let sessionStartTime = stream.session.startTime!
```

Comment:
I think that we don't set startTime in all cases. if we do then maybe better to force unwrap the value in declaration and not in all usages.